### PR TITLE
Newyork city todo Initial version

### PR DESCRIPTION
**Pull request by Govin-lala**
This pull request closes issue #21.

### CODE REVIEW

File: govin.mdNewyork.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Newyork ## 
+
+#### Restaurants 
+
```
Comment (Govin-lala) [inline govin.mdNewyork.md:4]: Add restaurants name here.

**Comment by llavelanet** [comment]
New York is great!

File: govin.mdNewyork.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+#### Things to do
+
```
Comment (Govin-lala) [inline govin.mdNewyork.md:6]: Just get there, you will get lots of things to explore

**Comment by Govin-lala** [comment]
👍

**Pull request merged**